assignment/api/src/resources/assignment.properties

```diff
@@ -1060,7 +1060,7 @@ sendFeedback.label = Send Feedback to Multiple Students
 sendFeedback.instruction = Select students from the table below. Write your feedback in the following text area, select any additional feedback options, and then click "Update" to save.
 sendFeedback.options = Feedback options
 sendFeedback.overwrite = Replace existing "Instructor Summary Comments" feedback for selected students.
-sendFeedback.returnToStudents = Return feedback to selected students now. (NOTE: This will release the feedback only, not grades.) 
+sendFeedback.returnToStudents = Return feedback to selected students now. (NOTE: Only the comments will be published, if you want to publish the grades you must click on "Release Grades").
```

Comment:
```suggestion
sendFeedback.returnToStudents = Return feedback to selected students now. (NOTE: Only the comments will be released, if you want to release the grades you must click on "Release Grades").
```

Reply:
Why don't we update the literals for ES and CA?

Reply:
@mpellicer I have made the changes in Transifex.